docs/source/deployment/azure_container_instance.rst

```diff
@@ -28,7 +32,12 @@ Prerequisites
 Deploying BentoService to Azure Container Instance
 --------------------------------------------------
 
-This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide <../quickstart>`:
+This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide <../quickstart>`.
```

Comment:
The context was explained in the first paragraph,  here simply say:

Run the example project form the `quick start guide <../quickstart>` to create the BentoML saved bundle for deployment:

Reply:
can you do the same for the other docs too?